pkg/apis/crunchydata.com/v1/doc.go

```diff
@@ -3,7 +3,6 @@ Crunchy PostgreSQL Operator API
 
 The Crunchy PostgreSQL Operator API defines HTTP(S) interactions with the Crunchy PostgreSQL Operator.
 
-
```

Comment:
:question: The version updates here are correct, but I'm wondering what caused the white space changes?

Reply:
Yeah, that's weird. Good eye! I'll back those out.